### PR TITLE
Throw an error when the provided path is not an asset catalog

### PR DESCRIPTION
**Pull request by djbe**
Title says it all.

### CODE REVIEW

File: swiftgen-cli/images.swift [hunk 1]
```diff
@@ -15,7 +15,13 @@ let imagesCommand = command(
   Argument<Path>("DIR", description: "Directory to scan for .imageset files.", validator: dirExists)
 ) { output, templateName, templatePath, enumName, path in
   let parser = AssetsCatalogParser()
-  parser.parseCatalog(at: path)
+
+  switch path.extension {
+  case "xcassets"?:
```
Comment (AliSoftware) [inline swiftgen-cli/images.swift]: Wouldn't an if/else be more simple here instead of switch with only one case?

File: swiftgen-cli/images.swift [hunk 2]
```diff
@@ -14,15 +14,13 @@ let imagesCommand = command(
   Option<String>("enumName", "Asset", flag: "e", description: "The name of the enum to generate"),
   Argument<Path>("DIR", description: "Directory to scan for .imageset files.", validator: dirExists)
 ) { output, templateName, templatePath, enumName, path in
-  let parser = AssetsCatalogParser()
-
-  switch path.extension {
-  case "xcassets"?:
-    parser.parseCatalog(at: path)
-  default:
+  guard let `extension` = path.extension, `extension` == "xcassets" else {
```
Comment (AliSoftware) [inline swiftgen-cli/images.swift]: Haha I was more thinking about `guard case "xcasstes" = path.extension` but that works too ^^

**Review by AliSoftware — APPROVED** [review]

File: swiftgen-cli/images.swift (hunk 2 above)
Comment (djbe) [inline swiftgen-cli/images.swift]: Ha, no, you're right, for some reason I automatically write unwrap code 😅

**Comment by AliSoftware** [comment]
Oh is Travis starting to *finally* waking up? (So frustrating to wait for the CI so long!)

**Comment by djbe** [comment]
Yeah, they probably had an enormous backlog of jobs to get through.

**Pull request merged**